requirements.txt

```diff
@@ -1,50 +1,71 @@
 asgiref==3.7.2
+attrs==23.2.0
+autobahn==23.6.2
+Automat==22.10.0
+awsebcli==3.20.10
+bcrypt==4.1.2
 black==24.2.0
 blessed==1.20.0
 boto==2.49.0
 boto3==1.34.73
-botocore==1.34.73
+botocore
```

Comment:
And also, could you add the version here? The build seems to be failing due to the versioning mismatch
`botocore==1.34.73`